Formula/j2objc.rb

```diff
@@ -0,0 +1,41 @@
+## TODO Prebuilt version, to be updated for build from source
+
+class J2objc < Formula
+  desc "Java to iOS Objective-C translation tool and runtime. (Prebuilt)"
+  homepage "http://j2objc.org"
+  url "https://github.com/google/j2objc/archive/1.3.1.zip"
+  sha256 "bd05db67fb233dfe7b58cefc899afb8f35099eae0e4b8a6486a398472fb2fac3"
+
+  def shim_script(target)
```

Comment:
Should be able to do something like this instead: https://github.com/Homebrew/homebrew-core/commit/a753bdca89177d120df006f18d0064b3fd3304d5